package.json

```diff
@@ -57,6 +57,7 @@
     "nativescript-webview-interface": "^1.4.2",
     "prismjs": "^1.15.0",
     "reflect-metadata": "~0.1.8",
+    "reveal-plugin-ios": "file:../reveal-plugin-ios",
```

Comment:
Remove this

Reply:
Not sure how this got in here (wasn't on purpose)... will remove